packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -20,6 +33,36 @@ def _ensure_routable():
     return _ensure_routable
 
 
+@retrying.retry(wait_fixed=2000,
```

Comment:
This was intentional. Unfortunately, while the cluster is coming up, it can throw assertions.